docs/latex.md

```diff
@@ -0,0 +1,13 @@
+# LaTeX Setup
+
+To contribue a scenario, you need the following as a minimum:
+
+- [**MiKTeX**](https://miktex.org/) for Windows, [**MacTeX**](https://www.tug.org/mactex/) for MacOS, [**TeX Live**](https://www.tug.org/texlive/) for Linux (required) to build the PDF file from LaTeX files
+- [**Inkscape**](https://inkscape.org/) to render glyphs in the document (while installing on Windows, make sure to tick `Add Inkscape to the System Path` option)
```

Comment:
We could add alternative/tip in case Inkscape doesn't do this - I didn't see this option in the installation wizard.

Reply:
Good idea, I'll split this per operating system.